### PR TITLE
Fix demo mode

### PR DESCRIPTION
**Pull request by atrovato**
- Fix some bad paths on demo mode
 - Delete confusing `demo.json` file
 - Add timer on DemoHttpClient to see loaders.

### CODE REVIEW

**Comment by Pierre-Gilles** [comment]
Oh shit I forgot to delete the demo.json file, my bad! 

Nice catch, I'm good for this!

File: front/src/utils/DemoHttpClient.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 export class DemoHttpClient {
   getDemoFile = async () => {
+    // Set a timer to see loaders
+    await new Promise(r => setTimeout(r, 200));
```
Comment (Pierre-Gilles) [inline front/src/utils/DemoHttpClient.js]: I'm good with that (feels more realistic), but 200ms seems too slow on my side, it looks like the demo page is slower than the real Gladys :D 

Could you change to 100ms ?

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]

File: front/src/utils/DemoHttpClient.js (hunk 1 above)
Comment (atrovato) [reply front/src/utils/DemoHttpClient.js]: I will add this as an "env" config, maybe disable it by default for public mode, but with env var, I can enable it on my side.
What do you think?

Comment (Pierre-Gilles) [reply front/src/utils/DemoHttpClient.js]: I'm good with that 👍

Comment (atrovato) [reply front/src/utils/DemoHttpClient.js]: Done, disabled by default, can be overridden with 
```
DEMO_MODE=true
DEMO_REQUEST_TIME=1000
```

Comment (Pierre-Gilles) [reply front/src/utils/DemoHttpClient.js]: Works on my side!

**Review by Pierre-Gilles — APPROVED** [review]

**Pull request merged**